src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -214,36 +218,13 @@ def test_run_app_dry_run(self, auth, c):
         "biokbase.narrative.jobs.appmanager.auth.get_agent_token",
         side_effect=mock_agent_token,
     )
-    def test_run_app_good_inputs(self, auth, c):
+    def test_run_app__good_inputs(self, auth, c):
         c.return_value.send_comm_message = MagicMock()
         new_job = self.am.run_app(
             self.test_app_id, self.test_app_params, tag=self.test_tag
         )
         self.assertIsInstance(new_job, Job)
-        expected_params = copy.deepcopy(self.test_app_params)
-        del expected_params["read_library_names"]
-        expected_params.update(
-            {
-                "read_library_refs": ["18836/5/1"],
-                "workspace_name": self.public_ws,
-                "assembler": None,
-                "pipeline": None,  # these get converted to None when params are checked
-            }
-        )
-        expected = {
-            "app_id": self.test_app_id,
-            "app_version": self.test_app_version,
-            "job_id": self.test_job_id,
-            "params": [expected_params],
-            "tag": self.test_tag,
-        }
-
-        for attr in JOB_ATTRS:
-            if attr in expected:
-                self.assertEqual(getattr(new_job, attr), expected[attr])
-            else:
-                self.assertEqual(getattr(new_job, attr), JOB_ATTR_DEFAULTS[attr])
-
```

Comment:
Kind of just deleted this. Didn't transplant this into the dry run test

Reply:
does `self.assertEqual(Object, Object)` do the same checks as these deleted code?

Reply:
It should do the right check for the params. It looks like what is lost is possibly testing the app version and tag. There's a bit of logic just to compute the "dry run" results so I was reluctant to get into this. If I try to patch up what is lost, I might end up just reverse engineering a lot of functionality and fortifying the "dry run" test

Reply:
Comprehensively tested the results of the "dry run" version of the method in question